### DISCUSSION TITLE
Δήλωση πρόθεσης παράτασης

### DISCUSSION THREAD

**Discussion by std159312** [sdy60/syllabus#134] [Q&A] [answered]
Καλησπέρα σας ύστερα από την σχετική συζήτηση που είχαμε θα ήθελα να μου δώσετε παράταση για την 4η Γραπτή Εργασία.

**Comment by Maurphal** [comment]
Και εγώ θα επιθυμούσα το ίδιο. Ευχαριστώ

**Comment by IoannisTheodoropoulos** [comment]
Καλησπέρα, θα ήθελα κι εγώ να αξιοποιήσω το δικαίωμα της παράτασης, ευχαριστώ!

**Comment by giorgos-christou** [comment]
Καλησπέρα, και εγω θα επιθυμούσα παράταση. Ευχαριστώ.

**Comment by gkalogeitonas** [comment]
Καλησπέρα, θα  ήθελα και εγώ να χρησιμοποιήσω την παράταση για την εργασία.

**Comment by epidrome** [comment, marked as answer]
έχει ενεργοποιηθεί η παράταση για όλους μέχρι 27 Μαρτίου, καλή δύναμη!

**Reply by std159312** [reply]
Καλησπέρα σας, σε εμένα φαίνεται ακόμα 20/3


![Screenshot_2024-03-14-16-54-43-295_com.android.chrome.jpg](https://github.com/sdy60/syllabus/assets/146677836/2bdc72b3-3624-4cbe-a59b-65f214c8202c)

**Reply by Maurphal** [reply]
και εμενα δεν έχει ενεργοποιηθεί η παραταση. μαλλον θα την υποβαλλω αυριο εκπροθεσμη και βλεπουμε.
![image](https://github.com/sdy60/syllabus/assets/146703365/f2f29f28-f1ca-4834-8b27-e8c844c395f8)

**Comment by iparthenios** [comment]
Θα επιθυμούσα κι εγώ την παράταση έως 27/03/24

Σας ευχαριστώ εκ των προτέρων!

**Comment by nikos7887** [comment]
Γεια σας. Θα επιθυμούσα και εγώ το δικαίωμα για παράταση.

Σας ευχαριστώ.

Και εμένα ακόμη φαίνεται στο σύστημα μέχρι 20/03.

**Comment by apapost** [comment]
Και εγώ θα επιθυμούσα παράταση μέχρι τις 27/03/2024

**Comment by Konssven** [comment]
Αν και πιστεύω ότι έχει ήδη απαντηθεί από τον κ Καθηγητή με το post παραπάνω θα ήθελα και εγώ παράταση για τις 27/03.
Και σε εμένα στο study έχει ημερομηνία παράδοσης 20/03. 
Πρέπει να ανεβάσουμε κάποιο αρχείο στο study μέχρι τις 20/03 ή θα αλλάξει ημερομηνία ; 
Ευχαριστώ

**Comment by std133524** [comment]
Δεν μπόρεσα να παραβρεθώ στις παρουσιάσεις/διαλέξεις λόγω σοβαρών υποχρεώσεων αλλά μπόρεσα τουλάχιστον να δημιουργήσω μια παρουσίαση. Πιστεύω είναι ενδιαφέρον για όποιον θέλει να την δεί. Καλή συνέχεια

https://github.com/std133524/g4/blob/main/g4_presentation_std133524(1).pdf
